resolving/npm-resolver/src/index.ts

```diff
@@ -350,7 +351,7 @@ function resolveFromLocalPackage (
       directory,
       type: 'directory',
     },
-    resolvedVia: 'local-filesystem',
```

Comment:
It doesn't seem like this field is used heavily in the rest of pnpm. I searched for `local-filesystem` in the repo and didn't see other code reading this in many places.